admin/adminActions.php

```diff
@@ -1248,6 +1253,28 @@ public function generateRegistrationLink(array $params)
 		return "Please give the user the following link: <br>".$thisURL.'?emailToken='.$emailToken;
 	}
 
+	public function changeReliability(array $params)
+	{
+		global $DB;
+
+		if (!isset($params['gameID']))
+		{
+			return "Please enter a game ID.";
+		}
+		
+		if (!isset($params['newRating']) || (!preg_match('/^[0-9][0-9]?$|^100$/', $params['newRating'])))
+		{
+			return "Please enter a valid reliability rating requirement (0-100).";
+		}
+
+		$gameID = (int)$params['gameID'];
+		$newRating = (int)$params['newRating'];
+
+		$DB->sql_put("UPDATE wD_Games SET minimumReliabilityRating = ".$newRating." WHERE id = ".$gameID);
+
+		return "Game ".$gameID."'s reliability rating has been set to ".$newRating;

```

Comment:
Can you turn this into a link to the game? And in the return message can you pull the old RR too so we can see in the logs what it used to be just in case for any debugging needed?